server/views/pages/reviewsTable.njk

```diff
@@ -107,4 +132,27 @@
 {% block bodyEnd %}
   {{ super() }}
   <script src="/assets/gaSendEvents.js"></script>
+
+  <script type="text/javascript" nonce="{{ cspNonce }}">
+    function hideBanner(event) {
+      const dismissLink = $(this)
+      const id = dismissLink.data('notification-id')
+
+      const notificationBanner = dismissLink.parents('.govuk-notification-banner')
+      const csrf = $('#_csrf').val()
+
+      $.ajax({
+        type: 'POST',
+        url: '/notification/dismiss',
+        headers: {
+          'X-CSRF-Token': csrf,
+        },
+        data: { id: id },
+      })
+        .done(() => notificationBanner.hide())
+        .fail(() => window.location = '/')
+    }
+
+    $('.notification_dismiss_link').on('click', hideBanner)
```

Comment:
i think this would be better to put into /assets/application.js

Reply:
Yeah that's "static" (without anything interpolated/rendered in it). I've moved to `application.js`: https://github.com/ministryofjustice/hmpps-incentives-ui/pull/362/commits/69a1708aae1e107388a1d930ef1e9b432cf2837c